.solhint.json

```diff
@@ -16,6 +16,7 @@
     "modifier-name-mixedcase": "warn",
     "no-empty-blocks": "off",
     "no-inline-assembly": "off",
+    "not-rely-on-time": "off",
```

Comment:
Because we rely on the EVM's block timestamp, if no timestamp is provided from reporter. See `reportResult(,,)` and `reportResultBatch` implementation on the `WitnetRequestBoardTrustableDefault` implementation.